feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test/bgp_override_as_path_split_horizon_test.go

```diff
@@ -441,6 +441,11 @@ func TestBGPOverrideASPathSplitHorizon(t *testing.T) {
 
 	t.Run("Configure DEFAULT network instance", func(t *testing.T) {
 		dutConfNIPath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut))
+		name := deviations.DefaultNetworkInstance(dut)
```

Comment:
use utility method to configure default network instance;

You can use fptest.ConfigureDefaultNetworkInstance

Reply:
Addressed this in the new commit